.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,32 +1,26 @@
-<!-- ^^^^^
-Please provide a concise, informative and self-explanatory title.
-Don't put issue numbers in there, do this in the PR body below.
-For example, instead of "Fixes #1234" use "Introduce new method to calculate 1+1"
--->
-<!-- Describe your changes here in detail -->
+<!-- ^ Please provide a concise and informative title. -->
+<!-- ^ Don't put issue numbers in the title, do this in the PR description below. -->
+<!-- ^ For example, instead of "Fixes #12345" use "Introduce new method to calculate 1 + 2". -->
+<!-- v Describe your changes below in detail. -->
+<!-- v Why is this change required? What problem does it solve? -->
+<!-- v If this PR resolves an open issue, please link to it here. For example, "Fixes #12345". -->
+
 
-<!-- Why is this change required? What problem does it solve? -->
-<!-- If this PR resolves an open issue, please link to it here. For example "Fixes #12345". -->
-<!-- If your change requires a documentation PR, please link it appropriately. -->
 
 ### :memo: Checklist
 
 <!-- Put an `x` in all the boxes that apply. -->
-<!-- If your change requires a documentation PR, please link it appropriately -->
-<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-<!-- Feel free to remove irrelevant items. -->
```

Comment:
Is there a strong reason why these two lines should be removed?

Reply:
That is about the checklist. 

First line: The items of the checklist seems clear enough. I doubt that someone wants to ask about them. 

Second line: No one needs to remove items from the checklist. They are checked or left unchecked.